cmd/devdata/system_intake.go

```diff
@@ -495,11 +495,12 @@ func createSystemIntakeDocument(
 ) *models.SystemIntakeDocument {
 	documentToCreate := &models.SystemIntakeDocument{
 		SystemIntakeRequestID: intake.ID,
-		CommonDocumentType:    models.SystemIntakeDocumentCommonTypeDraftICGE,
+		CommonDocumentType:    models.SystemIntakeDocumentCommonTypeDraftIGCE,
+		Version:               models.SystemIntakeDocumentVersionCURRENT,
 		FileName:              "create_and_get.pdf",
-		Bucket:                "bukkit",
-		S3Key:                 uuid.NewString(),
-		UploaderRole:          models.RequesterUploaderRole,
+		// Bucket:                "bukkit",
+		// S3Key:                 uuid.NewString(),
+		// UploaderRole:          models.RequesterUploaderRole,
```

Comment:
intended?

Reply:
Yes, the linter was yelling at me that these fields were unused. I went ahead and removed them here: [e423746](https://github.com/CMS-Enterprise/easi-app/pull/2748/commits/e423746421692fcd612abbe67965fa0ebff61bc5)